charts/longhorn/103.2.1+up1.5.3/questions.yaml.orig

```diff
@@ -0,0 +1,890 @@
+categories:
```

Comment:
This file is unnecessary. Can we make sure to cleanup and return the chart making follow https://confluence.suse.com/display/LON/How+to+create+Rancher+Chart+PRs

Reply:
Removed files and updated the confluence.
Thanks